dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ContentletIndexAPIImpl.java

```diff
@@ -200,6 +207,86 @@ private void addCustomMapping(final String indexName)  {
         addCustomMappingForRelationships(indexName, mappedRelationships);
     }
 
+    /**
+     * Builds a list of fields that have been marked to be "indexed" in dotCMS and tries to
+     * intelligently add the correct mapping for them based on their data type. The list of fields
+     * excludes fields that have custom mappings already defined for them. We build a list of structure
+     * inodes and then load them one by one, which will return them and their fields from cache.
+     * 
+     * @param indexName
+     * @throws ElasticsearchException
+     * @throws IOException
+     */
+    private void addStandardMappings(final String indexName) throws ElasticsearchException, IOException {
+
+        final ContentTypeAPI contentTypeAPI = APILocator.getContentTypeAPI(APILocator.systemUser());
+        final List<String> allTypes = Try.of(() -> new DotConnect().setSQL("select inode from structure").loadObjectResults())
+                        .getOrElseThrow(e -> new DotRuntimeException(e))
+                        .stream()
+                        .map(r -> (String) r.get("inode"))
+                        .collect(Collectors.toList());
+        final Map<String, Map<String, String>> mappingMap = new HashMap<>();
+        for (final String typeVar : allTypes) {
+            mappingMap.clear();
+            final ContentType type = Try.of(() -> contentTypeAPI.find(typeVar)).getOrNull();
+            if (type == null) {
```

Comment:
some warn or log here

Reply:
it runs on a full reindex, so a log would add too much unnecessary noise